content/maxwell3_fdem/sphere_freespace.rst

```diff
@@ -0,0 +1,262 @@
+.. _sphere_freespace:
+
+Response from a Conducting and Permeable Sphere in Free-Space
+-------------------------------------------------------------
+
+Introduction
+============
+
+Frequency-domain responses from ore bodies and unexploded ordnance items are of great practical importance to geophysicists.
+On a fundamental level, these responses can be understood by considering the excitation of a conductive and magnetically permeable sphere, under the influence of a harmonic inducing field.
+Here, expressions derived by Wait (1951) are used to characterize the excitation of the sphere in terms of an induced dipole moment.
+We examine how a target's dipole moment depends on its size and physical properties.
+A simple model for predicing the sphere's dipole response in free-space is then presented.
+For a sufficiently compact target, the free-space response represents a low-order approximation to the response of a conductive and permeable object within a resistive medium.
+
+
+Plane Wave Excitation of a Conducting and Magnetically Permeable Sphere
+=======================================================================
+
+In this section, equations derived by Wait (1951) are used to describe the induced magnetic dipole moment of a conductive and permeable sphere, resulting from excitation by a uniform magnetic field (Figure ??).
+According to Ward and Hohmann (1989), the frequency-domain wave equation within a homogeneous media can be expressed in terms of the magnetic Schelkunoff potential :math:`\vec F`:
+
+.. figure:: ./images/figGeometrySphere.png
+	:align: right
+        :scale: 45%
+        :name: GeometrySphere
+
+.. math::
+	\nabla^2 \vec F - \gamma^2 \vec F = 0
+	:label: Schelkunoff_Wave
+
+For a propagation media with conductivity :math:`\sigma`, magnetic permeability :math:`\mu` and electric permittivity :math:`\varepsilon`, the wavenumber :math:`\gamma` can be expressed as follows:
+
+.. math::
+	\gamma = \Big [ i \omega \mu \sigma - \omega^2 \mu \varepsilon \Big ]^{1/2}
```

Comment:
We would need to make sure this notation is consistent.
I think Mike, Sarah and I used "k" for the wavenumber.
Also the sign. I do not know which convention choose Sarah and Mike but for the MT I used "+k" in the wave equation (with the appropriate definition of k)